state/block_test.go

```diff
@@ -40,29 +38,111 @@ func TestGetBlockByNumber(t *testing.T) {
 
 	// Set the block's header & blockData in the blockState
 	// SetHeader also sets mapping [blockNumber : hash] in DB
-	err = stateService.Block.SetHeader(*blockHeader)
-	if err != nil {
-		t.Fatal(err)
-	}
+	err = stateService.Block.SetHeader(blockHeader)
+	require.Nil(t, err)
```

Comment:
Well, using require or not using is a matter of taste on tests, if people like it, it will be more adopted. There is no reason to change everything for it just yet IMHO.

Reply:
Ok, I don't think we should break consistency with how we have been writing it throughout the codebase. We should open and issue and switch format if we want to adopt this throughout the codebase but changing it here is not necessary and breaks consistency.

Reply:
Its test, no consistency is required. This is just how we can express the assertions, there are other ways to do it as well. If everytime we need to change everything to introduce a small improvement we will never be anywhere, always refactoring.

Reply:
I'm reluctant to comment here, since I see this conversation has been resolved.  I like the use of `require` (I was not aware of this package until now).  I agree it's a matter of taste, and it seems like a good idea.  I also agree that we should strive for consistency, it make the code base more readable/manageable.  However I don't think it's necessary to refactor existing working (and manageable), these are test cases, and both are readable (just that one way contains more lines).  Just IMHO.

Reply:
We should be consistent with the format of our tests. Not sure why "Its test, no consistency is required" applies here or where that rule should apply. If this is an improvement, we should make note to implement the improvement in all of our tests. The more consistent we are, the easier our code will be to read and maintain.

This may be an obvious change to you but it is less obvious to developers with less experience in Go and/or with our codebase.